wrangler.toml

```diff
@@ -1,6 +1,8 @@
 name = "cron-jobs"
 main = "src/worker.ts"
 compatibility_date = "2023-07-17"
+[triggers]
+crons = ["0 */4 * * *"]
```

Comment:
this will trigger our jobs to run every 4 hours.
This time is tentative and I am adding it as a part of scheduled event handler.
Will be changed post a proper discussion.